templates/layouts/default.jade

```diff
@@ -108,21 +107,16 @@ html
 				img.h1(src="/images/vector_faf_logo.png")
 				br
 				br
-				span: a(href='https://www.facebook.com/ForgedAllianceForever/') Facebook
-				span  | 
-				span: a(href='https://www.youtube.com/channel/UCkAWiUu4QE172kv-ZuyR42w') Youtube
-				span  | 
-				span: a(href='https://github.com/FAForever/') Github
-				span  | 
-				span: a(href='/contact') Contact Us
-				br
 				span: a(href='/keystone/signin') Admin Login
 				br
 				p Powered by <a href="http://keystonejs.com" target="_blank">KeystoneJS</a>.
 			.col-md-4: .text-center
-				h3 Unofficial Tools
-				a(href='http://fafafaf.bitbucket.org/') Replay Parser
-			
+				h3 Connect
+				div.social
+				<a href="https://www.facebook.com/ForgedAllianceForever" title="Follow FAForever on Facebook" target="_blank"><i class="fa fa-facebook fa-3x"></i></a>
```

Comment:
![screen shot 2016-07-14 at 1 53 02 pm](https://cloud.githubusercontent.com/assets/234712/16851703/5847aa28-49ca-11e6-889b-c853179c2efb.png)

Is that what you are looking for?